.eslintrc

```diff
@@ -38,6 +38,7 @@
     "newline-per-chained-call": ["warn", { "ignoreChainWithDepth": 5 }],
     "no-case-declarations": "warn",
     "no-confusing-arrow": ["warn", { "allowParens": true }],
+    "no-mixed-operators": 0,
```

Comment:
Don't change eslint rules.

Reply:
Hi @gilbarbara, I commonly never touch stuff outside the PR, but I got warnings for this missing rule in the code: for example at https://github.com/bopen/react-joyride/blob/39ed9e5f440363f99e038a2c70f27d458cac4bd9/src/components/Overlay.js#L156 or https://github.com/bopen/react-joyride/blob/39ed9e5f440363f99e038a2c70f27d458cac4bd9/src/components/Overlay.js#L163